index.js

```diff
@@ -125,7 +126,18 @@ var publicAgent = {
     if (!config.enabled) {
       return this;
     }
-    var logger = common.logger.create(config.logLevel, '@google/cloud-trace');
+
+    var logLevel = config.logLevel || 2;
```

Comment:
Is the specification of `logLevel` is not baked in? If not, I think having `logLevel` accept the name of the level makes more sense and I'd be all for it - though it would involve changing across the other agents as well to preserve uniformity.